recipes/gdk-pixbuf/all/conanfile.py

```diff
@@ -0,0 +1,118 @@
+from conans import ConanFile, Meson, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+import shutil
+
+
+class LibnameConan(ConanFile):
+    name = "gdk-pixbuf"
+    description = "toolkit for image loading and pixel buffer manipulation"
+    topics = ("conan", "gdk-pixbuf", "image")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://developer.gnome.org/gdk-pixbuf/"
+    license = "LGPL-2.1-or-later"
+    generators = "pkg_config"
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_libpng": [True, False],
+        "with_libtiff": [True, False],
+        "with_libjpeg": [True, False],
+        "with_jasper": [True, False],
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_libpng": True,
+        "with_libtiff": True,
+        "with_libjpeg": True,
+        "with_jasper": False,
+        }
+
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Macos":
+            # when running gdk-pixbuf-query-loaders
+            # dyld: malformed mach-o: load commands size (97560) > 32768
+            raise ConanInvalidConfiguration('This package does not support Macos currently')
+    
+    def build_requirements(self):
+        self.build_requires('meson/0.54.2')
+        if not tools.which('pkg-config') or self.settings.os == "Windows":
+            self.build_requires('pkgconf/1.7.3')
```

Comment:
```suggestion
        self.build_requires('pkgconf/1.7.3')
```

I prefer forcing the package to keep the reproducibility of this scenario.